sdk/python/arvados/arvfile.py

```diff
@@ -516,7 +517,7 @@ def _block_prefetch_worker(self):
                     return
                 self._keep.get(b)
             except Exception:
-                pass
+                _logger.error(traceback.format_exc())
```

Comment:
Apologies; I stand corrected. I wasn't aware that this method's behaviour was context sensitive.